### PR TITLE
Add AI configuration files and revamp instructions

### PR DESCRIPTION
**Pull request by amitjoshi-ms**
## Summary

Comprehensive revamp of AI configuration files based on research from [awesome-copilot](https://github.com/github/awesome-copilot) patterns.

### New Instruction Files (8)
| File | Purpose | ApplyTo |
|------|---------|---------|
| `typescript.instructions.md` | Type system, naming, patterns | `**/*.ts` |
| `testing.instructions.md` | Unit/E2E test structure | `tests/**/*.ts` |
| `game-logic.instructions.md` | Pure functions, state | `src/game/**/*.ts` |
| `ui.instructions.md` | DOM, accessibility, CSS | `src/ui/**/*.ts` |
| `security.instructions.md` | XSS, validation, storage | `**/*.ts, **/*.html` |
| `performance.instructions.md` | Optimization patterns | `**/*.ts, **/*.css` |
| `tooling.instructions.md` | Build tools, linters, configs | `*.config.*, *.json` |
| `ai-config.instructions.md` | Maintaining AI config files | `**/*.md` |

### Enhanced Prompts
- **start.task.prompt.md**: Added instruction references, task understanding workflow
- **review.task.prompt.md**: Added instruction references, streamlined checklist

### New Files
- **AGENTS.md**: Quick reference for AI coding agents

### Updated Files
- **copilot-instructions.md**: Added AI Configuration Files section indexing all instructions and prompts

## Testing
- ✅ `npm run typecheck` passes
- ✅ `npm run lint` passes
- ✅ `npm test` passes (77 tests)
- ✅ `npm run build` succeeds (13.22 KB)

### CODE REVIEW

File: .github/instructions/testing.instructions.md [hunk 1]
```diff
@@ -0,0 +1,197 @@
...
+      lastPlayer: 'O' as const,
+      expected: 'o-wins',
+    },
+  ];
+
+  testCases.forEach(({ name, board, lastPlayer, expected }) => {
+    it(`should detect ${name}`, () => {
+      expect(determineStatus(board, lastPlayer)).toBe(expected);
+    });
+  });
```
[the remark this answers is not in the document]
Comment (amitjoshi-ms) [reply .github/instructions/testing.instructions.md]: @copilot open a new pull request to apply changes based on [this feedback](https://github.com/amitjoshi-ms/gametime-tic-tac-toe/pull/54#discussion_r2680364052)

File: .github/prompts/review.task.prompt.md [hunk 2]
```diff
@@ -1,175 +1,87 @@
...
-3. Categorize findings by severity and type
-4. Provide specific, actionable feedback with line references
-5. Include code examples for recommended fixes
+### Performance
+- [ ] DOM updates batched where possible
+- [ ] No unnecessary re-renders
+- [ ] Event listeners properly managed
 
----
+### Branch Compliance
```
[the remark this answers is not in the document]
Comment (amitjoshi-ms) [reply .github/prompts/review.task.prompt.md]: @copilot open a new pull request to apply changes based on [this feedback](https://github.com/amitjoshi-ms/gametime-tic-tac-toe/pull/54#discussion_r2680364098)

File: .github/copilot-instructions.md [hunk 3]
```diff
@@ -1,28 +1,149 @@
-# gametime-tic-tac-toe Development Guidelines
+# Gametime Tic-Tac-Toe Development Guidelines
 
-Auto-generated from all feature plans. Last updated: 2026-01-10
+> Guidelines for GitHub Copilot and AI assistants working on this codebase.
```
[the remark this answers is not in the document]
Comment (amitjoshi-ms) [reply .github/copilot-instructions.md]: @copilot , this is updated by spec-kit tool and also manually. Can you fix the statement accordingly?

File: .github/copilot-instructions.md [hunk 4]
```diff
@@ -34,62 +155,118 @@ This project uses **Cloudflare Pages** with Git integration for automatic deploy
...
+
+This project uses structured AI configuration to provide consistent guidance.
+
+### Instructions (`.github/instructions/`)
+
+Focused rules auto-applied to specific file patterns via `applyTo`:
+
+| File | Purpose | Applies To |
+|------|---------|------------|
+| `typescript.instructions.md` | Type system, naming, patterns | `**/*.ts` |
```
[the remark this answers is not in the document]
Comment (amitjoshi-ms) [reply .github/copilot-instructions.md]: keep it.

File: .github/prompts/start.task.prompt.md [hunk 5]
```diff
@@ -1,98 +1,86 @@
...
-### Step 2: Create Feature Branch
+---
+description: Start new work with proper branch setup and task understanding
+mode: agent
+instructions:
+  - typescript.instructions.md
+  - game-logic.instructions.md
+  - testing.instructions.md
+  - ui.instructions.md
+---
```
[the remark this answers is not in the document]
Comment (amitjoshi-ms) [reply .github/prompts/start.task.prompt.md]: @copilot open a new pull request to apply changes based on [this feedback](https://github.com/amitjoshi-ms/gametime-tic-tac-toe/pull/54#discussion_r2680388563)

File: .github/copilot-instructions.md [hunk 6]
```diff
@@ -1,28 +1,149 @@
 # gametime-tic-tac-toe Development Guidelines
```
[the remark this answers is not in the document]
Comment (amitjoshi-ms) [reply .github/copilot-instructions.md]: outdated

File: .github/copilot-instructions.md [hunk 7]
```diff
@@ -1,28 +1,149 @@
...
 
-Auto-generated from all feature plans. Last updated: 2026-01-10
+> Guidelines for GitHub Copilot and AI assistants working on this codebase.
```
[the remark this answers is not in the document]
Comment (amitjoshi-ms) [reply .github/copilot-instructions.md]: outdated.

File: AGENTS.md [hunk 8]
```diff
@@ -0,0 +1,163 @@
...
+```
+
+## Architecture Rules
+
+### Separation of Concerns
+
+| Location | Responsibility | Side Effects |
+|----------|----------------|--------------|
+| `src/game/logic.ts` | Game rules | NONE - pure functions |
+| `src/game/state.ts` | State transitions | NONE - returns new objects |
```
[the remark this answers is not in the document]
Comment (amitjoshi-ms) [reply AGENTS.md]: @copilot open a new pull request to apply changes based on [this feedback](https://github.com/amitjoshi-ms/gametime-tic-tac-toe/pull/54#discussion_r2680395680)

File: .github/copilot-instructions.md [hunk 9]
```diff
@@ -34,62 +156,118 @@ This project uses **Cloudflare Pages** with Git integration for automatic deploy
...
+
+| File | Purpose | Applies To |
+|------|---------|------------|
+| `typescript.instructions.md` | Type system, naming, patterns | `**/*.ts` |
+| `testing.instructions.md` | Unit/E2E test structure | `tests/**/*.ts, **/*.test.ts, **/*.spec.ts` |
+| `game-logic.instructions.md` | Pure functions, state | `src/game/**/*.ts` |
+| `ui.instructions.md` | DOM, accessibility, CSS | `src/ui/**/*.ts` |
+| `security.instructions.md` | XSS, validation, storage | `**/*.ts, **/*.html` |
+| `performance.instructions.md` | Optimization patterns | `**/*.ts, **/*.css, **/*.html` |
+| `tooling.instructions.md` | Build tools, linters, configs | `*.json, *.config.ts, *.config.js, tsconfig.json, package.json` |
```
[the remark this answers is not in the document]
Comment (amitjoshi-ms) [reply .github/copilot-instructions.md]: @copilot open a new pull request to apply changes based on [this feedback](https://github.com/amitjoshi-ms/gametime-tic-tac-toe/pull/54#discussion_r2680423248)

File: .github/instructions/typescript.instructions.md [hunk 10]
```diff
@@ -0,0 +1,138 @@
+---
+description: 'TypeScript development standards for this project'
```
[the remark this answers is not in the document]
Comment (amitjoshi-ms) [reply .github/instructions/typescript.instructions.md]: @copilot open a new pull request to apply changes based on [this feedback](https://github.com/amitjoshi-ms/gametime-tic-tac-toe/pull/54#discussion_r2680457748)

File: .github/prompts/start.task.prompt.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (amitjoshi-ms) [reply .github/prompts/start.task.prompt.md]: @copilot open a new pull request to apply changes based on [this feedback](https://github.com/amitjoshi-ms/gametime-tic-tac-toe/pull/54#discussion_r2680457760)

File: .github/instructions/ui.instructions.md [hunk 11]
```diff
@@ -0,0 +1,220 @@
...
+        onCellClick(index);
+      }
+    });
+    container.dataset.boardClickBound = 'true';
+  }
+}
+```
+
+## Accessibility
+
```
[the remark this answers is not in the document]
Comment (amitjoshi-ms) [reply .github/instructions/ui.instructions.md]: @copilot open a new pull request to apply changes based on [this feedback](https://github.com/amitjoshi-ms/gametime-tic-tac-toe/pull/54#discussion_r2680480735)

File: .github/prompts/sync.repo.prompt.md [hunk 12]
```diff
@@ -1,6 +1,6 @@
-````prompt
 ---
 description: Sync local Git repository with remote, cleanup stale branches, and optimize storage
+mode: agent
 ---
```
[the remark this answers is not in the document]
Comment (amitjoshi-ms) [reply .github/prompts/sync.repo.prompt.md]: @copilot open a new pull request to apply changes based on [this feedback](https://github.com/amitjoshi-ms/gametime-tic-tac-toe/pull/54#discussion_r2680499347)

File: .github/instructions/tooling.instructions.md [hunk 13]
```diff
@@ -0,0 +1,226 @@
...
+  }
+}
+
+// ✅ Correct - dev dependency only
+{
+  "devDependencies": {
+    "typescript": "^5.9.3"
+  }
+}
+```
```
[the remark this answers is not in the document]
Comment (amitjoshi-ms) [reply .github/instructions/tooling.instructions.md]: @copilot open a new pull request to apply changes based on [this feedback](https://github.com/amitjoshi-ms/gametime-tic-tac-toe/pull/54#discussion_r2680493112)

[the remark this answers is not in the document]
Comment (amitjoshi-ms) [reply .github/instructions/tooling.instructions.md]: @copilot open a new pull request to apply changes based on [this feedback](https://github.com/amitjoshi-ms/gametime-tic-tac-toe/pull/54#discussion_r2680493112)

File: .github/instructions/ui.instructions.md [hunk 14]
```diff
@@ -0,0 +1,224 @@
...
+  };
+
+  container.addEventListener('click', handleClick);
+
+  // Caller is responsible for removing the listener when the board is torn down
+  return () => {
+    container.removeEventListener('click', handleClick);
+  };
+}
+```
```
[the remark this answers is not in the document]
Comment (amitjoshi-ms) [reply .github/instructions/ui.instructions.md]: @copilot open a new pull request to apply changes based on [this feedback](https://github.com/amitjoshi-ms/gametime-tic-tac-toe/pull/54#discussion_r2680493099)

File: .github/instructions/security.instructions.md [hunk 15]
```diff
@@ -0,0 +1,203 @@
...
+
+```typescript
+// ❌ Dangerous: XSS vulnerability
+container.innerHTML = `<span>${userInput}</span>`;
+
+// ✅ Safe: Use textContent for user data
+const span = document.createElement('span');
+// textContent treats userInput as plain text, preventing HTML interpretation and XSS
+span.textContent = userInput;
+container.appendChild(span);
```
[the remark this answers is not in the document]
Comment (amitjoshi-ms) [reply .github/instructions/security.instructions.md]: @copilot open a new pull request to apply changes based on [this feedback](https://github.com/amitjoshi-ms/gametime-tic-tac-toe/pull/54#discussion_r2680480780)

File: .github/copilot-instructions.md [hunk 16]
```diff
@@ -34,62 +156,122 @@ This project uses **Cloudflare Pages** with Git integration for automatic deploy
...
+### Instructions (`.github/instructions/`)
+
+Focused rules auto-applied to specific file patterns via `applyTo`:
+
+| File | Purpose | Applies To |
+|------|---------|------------|
+| `typescript.instructions.md` | Type system, naming, patterns | `**/*.ts, **/*.tsx` |
+| `testing.instructions.md` | Unit/E2E test structure | `tests/**/*.ts, **/*.test.ts, **/*.spec.ts` |
+| `game-logic.instructions.md` | Pure functions, state | `src/game/**/*.ts` |
+| `ui.instructions.md` | DOM, accessibility, CSS | `src/ui/**/*.ts, src/styles/**/*.css` |
```
[the remark this answers is not in the document]
Comment (amitjoshi-ms) [reply .github/copilot-instructions.md]: @copilot open a new pull request to apply changes based on [this feedback](https://github.com/amitjoshi-ms/gametime-tic-tac-toe/pull/54#discussion_r2680513491)

File: .github/instructions/ui.instructions.md (hunk 14 above)
[the remark this answers is not in the document]
Comment (amitjoshi-ms) [reply .github/instructions/ui.instructions.md]: @copilot open a new pull request to apply changes based on [this feedback](https://github.com/amitjoshi-ms/gametime-tic-tac-toe/pull/54#discussion_r2680523350)

File: .github/prompts/start.task.prompt.md [hunk 17]
```diff
@@ -1,98 +1,86 @@
...
+- If not: `git config core.hooksPath .githooks`
+
+### Sync and Create Branch
+```bash
+git stash                              # If uncommitted changes
+git checkout main
+git fetch origin && git pull origin main
+git checkout -b <prefix>-<name>        # e.g., feature-add-score
+git stash pop                          # If stashed
+```
```
[the remark this answers is not in the document]
Comment (amitjoshi-ms) [reply .github/prompts/start.task.prompt.md]: @copilot open a new pull request to apply changes based on [this feedback](https://github.com/amitjoshi-ms/gametime-tic-tac-toe/pull/54#discussion_r2680523363)

File: .github/instructions/game-logic.instructions.md [hunk 18]
```diff
@@ -0,0 +1,225 @@
...
+  // Cell must be empty
+  if (state.board[cellIndex] !== null) {
+    return false;
+  }
+  // Index must be valid
+  if (cellIndex < 0 || cellIndex > 8) {
+    return false;
+  }
+  return true;
+}
```
[the remark this answers is not in the document]
Comment (amitjoshi-ms) [reply .github/instructions/game-logic.instructions.md]: @copilot open a new pull request to apply changes based on [this feedback](https://github.com/amitjoshi-ms/gametime-tic-tac-toe/pull/54#discussion_r2680523368)

File: .github/instructions/game-logic.instructions.md [hunk 19]
```diff
@@ -0,0 +1,225 @@
...
+ */
+export function resetGame(): GameState {
+  return {
+    board: Array.from({ length: 9 }, () => null),
+    currentPlayer: 'X',
+    status: 'playing',
+    playerNames: loadPlayerNames(),
+  };
+}
+```
```
[the remark this answers is not in the document]
Comment (amitjoshi-ms) [reply .github/instructions/game-logic.instructions.md]: @copilot open a new pull request to apply changes based on [this feedback](https://github.com/amitjoshi-ms/gametime-tic-tac-toe/pull/54#discussion_r2680523343)

File: .github/instructions/ui.instructions.md [hunk 20]
```diff
@@ -0,0 +1,224 @@
...
+state.board.forEach((cell, index) => {
+  container.appendChild(createCell(cell, index)); // Reflow each time
+});
+```
+
+### Minimize Reflows
+
+- Read all DOM values first, then write
+- Use `classList` instead of setting `className` string
+- Avoid reading layout properties (`offsetHeight`) after writes
```
[the remark this answers is not in the document]
Comment (amitjoshi-ms) [reply .github/instructions/ui.instructions.md]: @copilot open a new pull request to apply changes based on [this feedback](https://github.com/amitjoshi-ms/gametime-tic-tac-toe/pull/54#discussion_r2680533456)

File: .github/instructions/performance.instructions.md [hunk 21]
```diff
@@ -0,0 +1,196 @@
...
+  renderBoard(container, state);
+  renderStatus(statusEl, state);
+  renderControls(controlsEl);
+}
+
+// ✅ Only update what changed
+function updateUI(prevState: GameState | null, newState: GameState) {
+  // NOTE: This relies on immutable updates: a new board array is created when the board changes.
+  // If you mutate the existing array in place, use a deeper comparison instead of reference equality.
+  if (!prevState || prevState.board !== newState.board) {
```
[the remark this answers is not in the document]
Comment (amitjoshi-ms) [reply .github/instructions/performance.instructions.md]: @copilot open a new pull request to apply changes based on [this feedback](https://github.com/amitjoshi-ms/gametime-tic-tac-toe/pull/54#discussion_r2680561517)

File: .github/instructions/tooling.instructions.md [hunk 22]
```diff
@@ -0,0 +1,225 @@
...
+  ...tseslint.configs.strictTypeChecked,
+  {
+    rules: {
+      '@typescript-eslint/no-unused-vars': ['error', { argsIgnorePattern: '^_' }],
+    },
+  },
+  { ignores: ['dist/', 'node_modules/', '*.config.*'] }
+);
+```
+
```
[the remark this answers is not in the document]
Comment (amitjoshi-ms) [reply .github/instructions/tooling.instructions.md]: @copilot open a new pull request to apply changes based on [this feedback](https://github.com/amitjoshi-ms/gametime-tic-tac-toe/pull/54#discussion_r2680561470)

File: .github/instructions/typescript.instructions.md [hunk 23]
```diff
@@ -0,0 +1,138 @@
...
+
+## Naming Conventions
+
+| Category | Convention | Example |
+|----------|------------|---------|
+| Types/Interfaces | PascalCase | `GameState`, `PlayerNames` |
+| Functions | camelCase | `handleCellClick`, `makeMove` |
+| Variables | camelCase | `gameState`, `currentPlayer` |
+| Constants | UPPER_SNAKE_CASE | `WINNING_LINES` |
+| Files | kebab-case | `game-logic.ts`, `player-names.ts` |
```
[the remark this answers is not in the document]
Comment (amitjoshi-ms) [reply .github/instructions/typescript.instructions.md]: outdated

File: .github/instructions/performance.instructions.md [hunk 24]
```diff
@@ -0,0 +1,201 @@
...
+  if (state.status !== 'playing') return state;
+  if (state.board[cellIndex] !== null) return state;
+
+  // Only proceed if move is valid
+  // NOTE: This pattern returns the same reference when the move is invalid,
+  //       allowing callers to skip re-renders by checking reference equality.
+}
+
+// Example caller using the reference-check contract:
+function onCellClick(currentState: GameState, cellIndex: number) {
```
[the remark this answers is not in the document]
Comment (amitjoshi-ms) [reply .github/instructions/performance.instructions.md]: @copilot open a new pull request to apply changes based on [this feedback](https://github.com/amitjoshi-ms/gametime-tic-tac-toe/pull/54#discussion_r2680588803)

File: .github/instructions/ui.instructions.md [hunk 25]
```diff
@@ -0,0 +1,235 @@
...
+      const indexStr = (cell as HTMLElement).dataset.index;
+      if (indexStr === undefined) return;
+      const index = parseInt(indexStr, 10);
+      onCellClick(index);
+    }
+  };
+
+  // Remove any existing listener to avoid duplicates
+  container.removeEventListener('click', handleClick);
+  container.addEventListener('click', handleClick);
```
[the remark this answers is not in the document]
Comment (amitjoshi-ms) [reply .github/instructions/ui.instructions.md]: @copilot open a new pull request to apply changes based on [this feedback](https://github.com/amitjoshi-ms/gametime-tic-tac-toe/pull/54#discussion_r2680588787)

File: .github/instructions/performance.instructions.md [hunk 26]
```diff
@@ -0,0 +1,219 @@
+---
+description: 'Performance optimization guidelines'
```
[the remark this answers is not in the document]
Comment (amitjoshi-ms) [reply .github/instructions/performance.instructions.md]: outdated.

File: .github/instructions/performance.instructions.md [hunk 27]
```diff
@@ -0,0 +1,201 @@
...
+  // still reference the same array, so the `prevState.board !== newState.board` check will not detect the change;
+  // in that case you must use a deeper comparison of the board contents instead of this shallow reference check.
+  // The board is re-rendered ONLY when its reference changes (i.e., when `prevState.board !== newState.board` evaluates to true).
+  // If only status changes (not the board), only renderStatus() is called, avoiding unnecessary work.
+  if (!prevState || prevState.board !== newState.board) {
+    updateBoard(container, newState);
+  if (!prevState || prevState.status !== newState.status) {
+    renderStatus(statusEl, newState);
+  }
+}
```
[the remark this answers is not in the document]
Comment (amitjoshi-ms) [reply .github/instructions/performance.instructions.md]: @copilot open a new pull request to apply changes based on [this feedback](https://github.com/amitjoshi-ms/gametime-tic-tac-toe/pull/54#discussion_r2680604857)

File: .github/instructions/testing.instructions.md [hunk 28]
```diff
@@ -0,0 +1,224 @@
...
+```
+
+## What NOT to Do
+
+- ❌ Write tests that depend on implementation details
+- ❌ Test private functions directly—test through public API
+- ❌ Leave `console.log` in tests
+- ❌ Use `any` types in test code
+- ❌ Write flaky tests that depend on timing
+- ❌ Skip writing tests for edge cases
```
[the remark this answers is not in the document]
Comment (amitjoshi-ms) [reply .github/instructions/testing.instructions.md]: @copilot open a new pull request to apply changes based on [this feedback](https://github.com/amitjoshi-ms/gametime-tic-tac-toe/pull/54#discussion_r2680609052)

File: .github/instructions/tooling.instructions.md [hunk 29]
```diff
@@ -0,0 +1,230 @@
...
+- Use SVG icons (scalable, small file size)
+- `display: standalone` for app-like experience
+
+## Common Mistakes
+
+- Editing generated or vendor files instead of the primary config (e.g., changing `node_modules` contents instead of `package.json` or tool configs).
+- Introducing invalid JSON syntax in `package.json` or `manifest.json` (trailing commas, comments, or single quotes). Remember: only JSONC examples in this doc support comments, not actual config files.
+- Letting `tsconfig.json` and tooling configs (ESLint, Vite, Vitest, Playwright) drift out of sync—for example, adding a new `src/` subfolder in `tsconfig.json` but forgetting to update related tool config if it uses explicit paths.
+- Renaming or removing npm scripts in `package.json` without updating CI, documentation, or local dev instructions that depend on those script names.
+- Ignoring config-specific `ignores`/`include` patterns (e.g., failing to exclude `dist/` or `node_modules/`), which can slow down tooling or cause spurious lint/test results.
```
[the remark this answers is not in the document]
Comment (amitjoshi-ms) [reply .github/instructions/tooling.instructions.md]: @copilot open a new pull request to apply changes based on [this feedback](https://github.com/amitjoshi-ms/gametime-tic-tac-toe/pull/54#discussion_r2680609054)

File: .github/copilot-instructions.md [hunk 30]
```diff
@@ -34,62 +156,122 @@ This project uses **Cloudflare Pages** with Git integration for automatic deploy
...
+
+| Prompt | Mode | Use When |
+|--------|------|----------|
+| `start.task.prompt.md` | Agent | Starting new work (creates branch, understands task) |
+| `review.task.prompt.md` | Agent | Pre-commit review (quality, security, tests) |
+| `sync.repo.prompt.md` | Agent | Syncing local work with `main`/remote and updating branches |
+| `submit.task.prompt.md` | Agent | Preparing changes for PR/merge (summaries, checklists, metadata) |
+| `release.latest.prompt.md` | Agent | Working on the latest release (notes, checks, deployment steps) |
+
+**Usage:** Common commands: `/start`, `/review`, `/sync`, `/submit`, `/release` in Copilot Chat, or reference via `@workspace /start` (and analogous names).
```
[the remark this answers is not in the document]
Comment (amitjoshi-ms) [reply .github/copilot-instructions.md]: @copilot , correct prompts are  `sync.repo`, `/start.task`, `/review.task`, `/submit.task`, `release.latest`

File: .github/instructions/performance.instructions.md [hunk 31]
```diff
@@ -0,0 +1,202 @@
...
+function debounce<T extends (...args: unknown[]) => void>(
+  fn: T,
+  delay: number
+): (...args: Parameters<T>) => void {
+  let timeoutId: number | undefined;
+  return (...args) => {
+    clearTimeout(timeoutId);
+    timeoutId = window.setTimeout(() => fn(...args), delay);
+  };
+}
```
[the remark this answers is not in the document]
Comment (amitjoshi-ms) [reply .github/instructions/performance.instructions.md]: @copilot open a new pull request to apply changes based on [this feedback](https://github.com/amitjoshi-ms/gametime-tic-tac-toe/pull/54#discussion_r2680612530)

File: .github/instructions/tooling.instructions.md [hunk 32]
```diff
@@ -0,0 +1,230 @@
...
+  },
+  resolve: {
+    alias: { '@': resolve(__dirname, 'src') }    // Must match tsconfig.json
+  },
+});
+```
+
+**Rules:**
+- You may exclude orchestration-only entrypoints (for example, `src/main.ts`)
+- Do not exclude core game logic files (e.g., `src/game/**/*.ts`)
```
[the remark this answers is not in the document]
Comment (amitjoshi-ms) [reply .github/instructions/tooling.instructions.md]: @copilot open a new pull request to apply changes based on [this feedback](https://github.com/amitjoshi-ms/gametime-tic-tac-toe/pull/54#discussion_r2680626287)
